library/shellwords/shellwords_spec.rb

```diff
@@ -26,4 +26,11 @@
   it "raises ArgumentError when single quoted strings are misquoted" do
     lambda { shellwords("a 'b c d e") }.should raise_error(ArgumentError)
   end
+
+  ruby_version_is '2.4' do
+    # https://bugs.ruby-lang.org/issues/10055
+    it %Q{ treats the backslash as escape character only when followed by one of the following characters: $ ` " \ } do
```

Comment:
This description has extra spaces at the beginning and end, could you remove them?
I think single quotes would work fine here (`'`).

Reply:
@eregon done